### PR TITLE
completed

### PR DESCRIPTION
**Pull request by usama-in**

### CODE REVIEW

File: src/App.css [hunk 1]
```diff
@@ -0,0 +1,12 @@
+/* In your global CSS file (e.g., index.css) */
```
Comment (abdulhananasif) [inline src/App.css:1]: remove comment

File: src/App.jsx [hunk 2]
```diff
@@ -1,14 +1,34 @@
...
+import CreateTodo from "./pages/CreateTodo";
+import EditTodo from "./pages/EditTodo";
+import Root from "./Root";
+import ErrorPage from "./pages/ErrorPage";
 
 function App() {
+
+  //configure router using version 6
   
+  const router = createBrowserRouter( [
```
Comment (abdulhananasif) [inline src/App.jsx]: extract this to separate file named constants or something similar

File: src/Root.jsx [hunk 3]
```diff
@@ -0,0 +1,16 @@
+import React from "react";
+import Header from "./components/Header";
+import { Outlet } from "react-router-dom";
+
+const Root = () => {
+  return (
+    <div>
+      <Header />
+      <main className="mt-[2rem] ml-auto mr-auto w-[60vw]">
```
Comment (abdulhananasif) [inline src/Root.jsx]: mt-[2rem] and w-[60vw] should be added to tailwind config

File: src/components/FormComponent.jsx [hunk 4]
```diff
@@ -0,0 +1,145 @@
...
+  //nevigate to main page
+  const handleCancel = (event) => {
+    event.preventDefault();
+    navigate("/todos");
+  };
+
+  return (
+    <div>
+      <form className="max-w-lg mx-auto p-6 bg-background text-text-color rounded-lg shadow-md">
+        <h1 className="text-center mb-5 text-[18px] font-bold">
```
Comment (abdulhananasif) [inline src/components/FormComponent.jsx]: add font size in tailwind config

File: src/components/Header.jsx [hunk 5]
```diff
@@ -0,0 +1,30 @@
...
+  const day = date.getDate();
+
+  //this will get month in string e.g(January)
+  const month = date.toLocaleString("default", { month: "long" });
+  const year = date.getFullYear();
+  console.log(day, month, year);
+
+  return (
+    <div className="w-screen">
+      <header className="bg-background w-[100vw] h-auto text-text-color flex flex-col justify-center items-center ">
```
Comment (abdulhananasif) [inline src/components/Header.jsx:16]: w-[100vw] is already in tailwind classes under the name w-screen

File: src/components/Header.jsx [hunk 6]
```diff
@@ -0,0 +1,30 @@
...
+        <p className="text-[22px] md:text-[30px]">
```
Comment (abdulhananasif) [inline src/components/Header.jsx]: add font size in tailwind config

File: src/components/Todos.jsx [hunk 7]
```diff
@@ -0,0 +1,62 @@
...
+   
+    const updatedTodosAfterDelete= toDos.filter(item => item.id != id)
+
+    setToDos(updatedTodosAfterDelete)
+    localStorage.setItem('todos' , JSON.stringify(updatedTodosAfterDelete))
+    
+  }
+
+  
+console.log(toDos)
```
Comment (abdulhananasif) [inline src/components/Todos.jsx]: remove console

**Review by abdulhananasif — CHANGES_REQUESTED** [review]

File: src/App.jsx (hunk 2 above)
Comment (usama-in) [reply src/App.jsx]: done

File: src/Root.jsx (hunk 3 above)
Comment (usama-in) [reply src/Root.jsx]: done

**Review by usama-in — COMMENTED** [review]
everything done.

**Pull request merged**